massgov/mayflower
# Patternlab/dp 14006  update manage your account

## Description
Update "Manage your account" stickyNav id

## Related Issue / Ticket

- [DP-14006](https://jira.mass.gov/browse/DP-14006)

## Steps to Test


1. from the /?p=organisms-account-manager pattern check that the H3 id="manage-your-account"
https://mayflower.digital.mass.gov/b/patternlab/DP-14006--update-manage-your-account-id/patterns/03-organisms-by-author-account-manager/03-organisms-by-author-account-manager.html

## Screenshots
![Screen Shot 2019-05-20 at 10 10 23 AM](https://user-images.githubusercontent.com/18662734/58035567-ac1e9000-7aee-11e9-8942-efdc72e60639.png)


## Note:
This PR fixes a sticky nav issue encountered while Melissa was working on [DP-10836](https://jira.mass.gov/browse/DP-10836)
